courses-app/src/index.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
```

Comment:
Best practice.

Here and in all other files
Add empty strings between import groups.

import from '[node_modules]'

import from '[own_components]'
import from '[constants]'

import from '[styles]'